db/migrate/20220728193602_fix_doc_sentencing_counties_id.rb

```diff
@@ -0,0 +1,6 @@
+class FixDocSentencingCountiesId < ActiveRecord::Migration[6.0]
+  def change
+    rename_column :doc_sentences, :doc_sentencing_counties_id, :doc_sentencing_county_id
```

Comment:
I'm a little confused, are these different migration files wrong or bad? I thought It was good to have different migration files instead of one big one? plus If I remember correctly  this migration file is new to the project since its dealing with a new table. are you saying I should revert all my migrations that I've done to fit it into one file?

Reply:
I'd have one migration file per commit per table you're working on

Reply:
Not a big deal though, just something nice for future to keep migrations a bit cleaner

Reply:
☝️ I agree with @sgelbart. We can just rollback and make this change on the original migration. We can pair on it on Friday

Reply:
Sounds good!